server/models/message.py

```diff
@@ -0,0 +1,119 @@
+
+from datetime import datetime
+from sqlalchemy import (
+    Column, String,
+    Integer, ForeignKey,
+    Text, Boolean,
+    DateTime, and_
+    )
+from sqlalchemy.orm import relationship, validates
+from models.base_model import Base, BaseModelMixin
+from db import session
+
+class Message(Base, BaseModelMixin):
+    """
+    sender - The user that originally sent the message
+
+    receiver - The user that the message was intended for
+    
+    file_url - In case we do need to have file sending, this
+    can be used to store the url from s3
+
+    content - The actual content of the message
+
+    created_at - When the message was sent/created. This will
+    not really be used except in the database/programatically
+    for sorting/ordering to ensure proper display.
+    Everywhere else, the return of 'get_formatted_date_time'
+    will be used instead
+    """
+    __tablename__ = "message"
+
+    id = Column(Integer, primary_key=True)
+    # Foreign Keys
+    conversation_id = Column(Integer, ForeignKey("conversation.id"))
+    sender_id = Column(Integer, ForeignKey('user.id'))
+    receiver_id = Column(Integer, ForeignKey('user.id'))
```

Comment:
So, are you saying that we would instead do a check on the Conversation instance for the user that isn't the current one? So should I put a method in Conversation, something like `get_receiver` which takes in the current users id and returns the user that doesn't match the given id? I'm suggesting the method because there would more than likely be repeated code without it.